command/connect/redirecttraffic/redirect_traffic.go

```diff
@@ -48,6 +54,14 @@ func (c *cmd) init() {
 	c.flags.IntVar(&c.proxyInboundPort, "proxy-inbound-port", 0, "The inbound port that the proxy is listening on.")
 	c.flags.IntVar(&c.proxyOutboundPort, "proxy-outbound-port", iptables.DefaultTProxyOutboundPort,
 		"The outbound port that the proxy is listening on. When not provided, 15001 is used by default.")
+	c.flags.Var((*flags.AppendSliceValue)(&c.excludeInboundPorts), "exclude-inbound-port",
+		"Inbound port to exclude from traffic redirection. May be provided multiple times.")
+	c.flags.Var((*flags.AppendSliceValue)(&c.excludeOutboundPorts), "exclude-outbound-port",
+		"Outbound port to exclude from traffic redirection. May be provided multiple times.")
+	c.flags.Var((*flags.AppendSliceValue)(&c.excludeOutboundCIDRs), "exclude-outbound-cidr",
+		"Outbound CIDR to exclude from traffic redirection. May be provided multiple times.")
+	c.flags.Var((*flags.AppendSliceValue)(&c.excludeUIDs), "exclude-uid",
```

Comment:
Would it be valuable to also be able to exclude GIDs?

Reply:
I think it could be. The main reason we're adding this is to better support the VM use case when you might want to exclude other processes/uids/gids from traffic redirection. I'm inclined to wait for users' feedback on this one because I'm not sure what would be valuable there.